libs/application/templates/driving-license/src/forms/application.tsx

```diff
@@ -276,7 +315,8 @@ export const application: Form = buildForm({
             buildDividerField({}),
             buildKeyValueField({
               label: m.overviewTeacher,
-              value: ({ answers: { teacher } }) => teacher as string,
+              value: 'Arnór Heiðar Sigurðsson',
```

Comment:
Is this maybe something used in development, should this maybe be?
```suggestion
              value: ({ answers: { teacher } }) => teacher as string
```